docs/source/installation.rst

```diff
@@ -39,20 +39,35 @@ There are fundamentally two different approaches to install ``IvadoMed``:
 Approach 1: Conda
 ------------------
 
-Step 1: Create new Conda Env called IvadoMedEnv (may taken 5 to 15 minutes)
+Step 1: Clone the `ivadomed <https://github.com/ivadomed/ivadomed>`_ repository.
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+::
+
+    git clone https://github.com/neuropoly/ivadomed.git
+    cd ivadomed
+
+Step 2: Create new Conda Env called IvadoMedEnv (may taken 5 to 15 minutes)
 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
 ::
 
     conda env create --file environment.yml
 
-Step 2 : Activate environment and use
+Step 3 : Activate environment and use
 ++++++++++++++++++++++++++++++++++++++
 
 ::
 
     conda activate IvadoMedEnv
 
+Step 4 : Install from source
+++++++++++++++++++++++++++++++++++++++
+
+::
+
+    pip install -e .
+
 Note that this is NOT compatible with ComputeCanada because of their no anaconda policy: https://docs.computecanada.ca/wiki/Anaconda/en
```

Comment:
i would remove-- i would minimize the references to CC-- ivadomed is an open source project dedicated to the international community-- people outside of canada have no clue what CC is so it gives the impression from "outsiders" that they are penetrating a closed circle and we don't want to give that impression-- we want to be inclusive